ecosystem/typescript/sdk/src/move_scripts/token_transfer_with_opt_in/sources/scripts.move

```diff
@@ -0,0 +1,17 @@
+script {
+    use aptos_token::token::{Self, transfer};
+    use std::string::String;
+
+    fun main(
+        from: &signer,
+        creator: address,
+        collection_name: String,
+        token_name: String,
+        token_property_version: u64,
+        to: address,
+        amount: u64,
+    ) {
+        let token_id = token::create_token_id_raw(creator, collection_name, token_name, token_property_version);
+        transfer(from, token_id, to, amount);
+    }
+}
```

Comment:
what if python SDK wants to call this script as well? Can move this script out of the SDK scope? Also, some comments will be helpful.

Reply:
this is temporary. After mainnet release, we should be able to delete this script.

Reply:
These files will be packaged and get published to NPM. Therefore I still prefer to move this out.

Reply:
sounds good, I can just keep the bytecode then